mobile/lib/features/trade/trade_screen.dart

```diff
@@ -91,10 +101,15 @@ class TradeScreen extends StatelessWidget {
                 child: TradeTabs(
                   tabs: const [
                     "Positions",
+                    "Trades",
```

Comment:
🙃 How about Positions, Orders, Trades.. Don't know why but this seems more logical to me 🙈

Reply:
I can flip that.